src/main/java/pl/cleankod/ApplicationInitializer.java

```diff
@@ -73,4 +79,17 @@ AccountController accountController(FindAccountAndConvertCurrencyUseCase findAcc
     ExceptionHandlerAdvice exceptionHandlerAdvice() {
         return new ExceptionHandlerAdvice();
     }
+
+    @Bean
+    public SimpleModule singleValueObjectModule() {
+        SimpleModule module = new SimpleModule();
+        module.addSerializer(SingleValueObject.class, new JsonSerializer<>() {
```

Comment:
Does this correctly work with the generics that are used in the `SingleValueObject` class?